### PR TITLE
backupd, corebrightnessd, fontd, translationd, warmd: add page

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

For #6596

### CODE REVIEW

**Review by blueskyson — COMMENTED** [review]
Please add more-information-link to each page. Links from www.unix.com, ss64.com, www.manpagez.com are accepted for osx.

**Comment by meowmeowmeowcat** [comment]
FYI, `translationd` has no man page on the above sites.

File: pages/osx/backupd.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# backupd
+
+> The Time Machine backup daemon.
+> It is responsible for creating Time Machine backups and managing its backup history.
```
Comment (blueskyson) [inline pages/osx/backupd.md:4]:
```suggestion
> Create Time Machine backups and manage its backup history.
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages/osx/backupd.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# backupd
+
+> Create Time Machine backups and manage its backup history.
```
Comment (pixelcmtd) [inline pages/osx/backupd.md:3]:
```suggestion
> Creates Time Machine backups and manages its backup history.
```

**Review by pixelcmtd — APPROVED** [review]
LGTM otherwise.

**Pull request merged**